.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -6,11 +6,19 @@ title: 🚀 [Feature]
 ---
 
 ## :rocket: Feature
-<!-- Describe the Feature. Use Screenshots if possible. -->
+<!-- Give a short summary of the Feature. Use Screenshots if you want. -->
+
+### User Problem
+<!-- Which problem is this solving? Why do you think this is important? Who will benefit from it and how? -->
+
+### Implementation
+<!-- How do you think this feature should be implemented? How will it be used? Where in the network should it be located? Which steps and screens are involved? -->
 
 ### Design & Layout
-<!-- Attach Screenshots and Drawings. -->
+<!-- Attach Screenshots and Sketches to illustrate your idea. -->
 
+### Validation
+<!-- How can we make sure that this feature indeed solves the above problem? How do we know if it has been accepted by the users of the network, once released? -->
 
 ### Additional context
-<!-- Add any other context or screenshots about the feature request here.-->
+<!-- Add other context or background about the feature request here.-->
```

Comment:
code creep! :stuck_out_tongue: 
we try to avoid making unrelated changes to `PR`s... it takes longer for `PR` reviews and is just generally bad practice...
Could you please put in a separate `PR` with these changes? We could also just look the other way in this case, but we want to build a culture of avoiding this.


Acutally, having a closer look, maybe this wasn't added by you? the commit history looks all jumbled up... could you please rebase this `PR` to `master` so that only the commits relevant to this `PR` appear?